alertmanager_gchat_integration/app.py

```diff
@@ -70,10 +70,11 @@ def post_alerts():
         text_alert = J2_TEMPLATE_ENGINE.render(render_payload)
```

Comment:
Consider maybe to rename the `text_alert` variable to `rendered_alert` to avoid confusion.